Kitodo/src/test/resources/hibernate.cfg.xml

```diff
@@ -36,20 +36,23 @@
         <property name="hbm2ddl.auto">create-drop</property>
 
         <!-- Die einzelnen Mappings -->
-        <mapping class="org.kitodo.data.database.beans.Authorization"/>
+        <mapping class="org.kitodo.data.database.beans.Authority"/>
         <mapping class="org.kitodo.data.database.beans.Batch"/>
+        <mapping class="org.kitodo.data.database.beans.Client"/>
         <mapping class="org.kitodo.data.database.beans.Docket"/>
         <mapping class="org.kitodo.data.database.beans.Filter"/>
         <mapping class="org.kitodo.data.database.beans.History"/>
         <mapping class="org.kitodo.data.database.beans.LdapGroup"/>
         <mapping class="org.kitodo.data.database.beans.LdapServer"/>
         <mapping class="org.kitodo.data.database.beans.Process"/>
-        <mapping class="org.kitodo.data.database.beans.Project"/>
```

Comment:
Why did you remove this line and add it two lines later? You did this not in the main and selenium version of this file.

Reply:
Because the mappings were not indentical in every file. They are identical with this change.

Reply:
Then you changed a alphabetical sorted to a not so sorted list :)

Reply:
Ok, to prevent that hell opens its doors, i will ifx that ;-)